README.md

```diff
@@ -58,7 +58,29 @@ being rate-limited first-hand!
 
 ### 2. Getting your credentials from GitHub
 
-You will need to register an OAuth application with GitHub by following these steps:
+There are two approaches to getting credentials from GitHub.
+First, you can get an access token. Second, you can register an OAuth app.
+The second approach is not recommended, and will be removed in a future release.
+
+#### Getting an access token
+
+This is the recommended way to authenticate yourself with GitHub.
```

Comment:
If you add in the title, you can strike this sentence if desired.

Reply:
:+1: